README.md

```diff
@@ -17,6 +17,21 @@ This library currently includes the following charts:
 
 Note: This library is still in beta, please use carefully, and file issues as discovered. Pull requests for additional charts always welcome!
 
+## Install
+First install `ember-sparkes` to your application:
+```bash
+ember install ember-sparkes
+```
+Then add default `ember-resize` configuration into `config/environment.js` file:
```

Comment:
Then add default `ember-resize` configuration into `config/environment.js` file, these properties can be changed to suit your needs. Please see the documentation for [ember-resize](https://github.com/mike-north/ember-resize) for more information: